### ISSUE TITLE
9ED/10E/SHM/LRW/MOR localized in Traditional Chinese

### ISSUE THREAD

**Issue by csuhta** [scryfall/gatherer-bugs#45]
The following sets are localized on Gatherer in Traditional Chinese, but no such cards physically exist:

Ninth Edition
Tenth Edition
Shadowmoor
Lorywn
Morningtide

**Comment by csuhta** [comment]
@hongalex Yes! Thanks for pointing that out. That was an erroneous inclusion here

**Comment by a-tangible-human-skull** [comment]
Still the case in Gatherer '25.

https://gatherer.wizards.com/search?setName=eq~Ninth_Edition~9ED%2Ceq~Tenth_Edition~10E%2Ceq~Shadowmoor~SHM%2Ceq~Lorwyn~LRW%2Ceq~Morningtide~MOR&language=eq~Chinese%20Traditional~zh-tw&page=16

They even include new digital renders for the cards, despite not existing in paper.
